blocks.js

```diff
@@ -2599,7 +2599,7 @@ BlockMorph.prototype.restoreInputs = function (oldInputs) {
 
 BlockMorph.prototype.showHelp = function () {
     var myself = this,
-        ide = this.parentThatIsA(IDE_Morph),
+        ide = this.parentThatIsA(BlockEditorMorph).siblings()[0],
```

Comment:
Ugh, I don't why I didn't try that...but you're right! 

Your solution definitely seems like it should work.